README.md

```diff
@@ -55,7 +55,7 @@ digraph G {
 
 which could be used with tools like [Webgraphviz](http://www.webgraphviz.com/) to get a graphic like
 
-![image](https://user-images.githubusercontent.com/4211715/52170132-a8971100-276a-11e9-8ef3-b4cfdb494fb6.png)
+![image](/images/webgraphimage_1.png?raw=true "Optional Title")
```

Comment:
Two suggestions here! 

1. Rename your image name to `graphviz.png`
1. Remove `?raw?=true "Optional Title"` 

The reason for  2nd suggestion would be if we just have `/images/graphviz.png` instead of `/images/graphviz.png?raw=true`, it would be treated as a file path from current directory in most of the text editors which support markdown rendering and also by markdown readers. Thus it would be good for users trying to read the README.md from their local copies.